forte_wrapper/allennlp/allennlp_processors.py

```diff
@@ -116,6 +117,8 @@ def default_configs(cls):
                 multiple models are loaded, cuda devices are assigned in a
                 round robin fashion. E.g. [0, -1] -> first model uses gpu 0
                 but second model uses cpu.
+            - infer_batch_size: batch size used for model inference.
+                A value <= 0 means no limit.
```

Comment:
I guess we need to say the size is corresponding to `Sentence` in this case?

Reply:
Re-phrased it to: maximum number of sentences passed in as a batch to model's predict function.